tutorial_plugin_java.md

```diff
@@ -0,0 +1,154 @@
+﻿---
+layout: page
+title: Tutorial Plugin Java
+permalink: /plugin/tutorial/java
+---
+
+# How to create a plugin for SmartSHARK
```

Comment:
"This repository" does not work here.

Maybe "This tutorial exaplins how to create Jjava plugin for SmartSHARK"?